survey_crm_generation/static/tests/survey_crm_generation_tour.js

```diff
@@ -0,0 +1,60 @@
+odoo.define(
```

Comment:
The tests do not have the correct structure. The js files in v17 should have the extension ‘.esm.js’ and have the tag ‘/** @odoo-module */’ in the header, also tours are now done in registry.category(‘web_tour.tours’). Please review the tours and adapt them to run in v17.